### ISSUE TITLE
A11Y - voice over focus is moved to the address bar and the success message "Card saved" is not announced for remember card flow

### ISSUE THREAD

**Issue by nbhasin2** [mozilla-mobile/firefox-ios#15324]
## Steps to reproduce
* Foxfooding Bug: 6/30/2023 16:03:35
* No particular steps
* Foxfooding Notes
* Try to perform a remember your card flow for credit card
* (a11y S2, build 9000) When the card is saved and a status bar is added to the bottom of the page, the VoiceOver focus is moved to the address bar and the success message "Card saved" is not announced to a VoiceOver user.
-- 1.  While moving focus to the status bar message is not required, it should be communicated to the assistive technology by posting a notification or a queued announcement.
-- 2. After any drawer is closed, it's expected that the focus is either moved to the same element that triggered the drawer of, if this element is no longer available, like in this case, to the top of the updated content - in this case, the focus is expected to land on the beginning of the web content (instead of the address bar, because the user would have to navigate past the repetitive to them content of the address bar and toolbar while trying to get back to the web page)

### Expected behavior
* See points 1 and 2 above for how the message should be announced 

### Actual behavior
*  the VoiceOver focus is moved to the address bar and the success message "Card saved" is not announced to a VoiceOver user.

### Device & build information
* Device: iPhone
* Build version: 32291
* First seen version: ? 
### Notes
Attachments: 
![](https://github.com/mozilla-mobile/firefox-ios/assets/8919439/11c088b2-fdc2-4e0c-b24a-63863c63608a)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-6883)

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

I tried to verify this issue on v122 (37175) with iPhone 15 Pro (17.1.2) but I do believe we still have some issues.
New card Saved - is not fully said only New Card. 

Web content  followed by Reader Mode available said 2x and after that, I can navigate in all the page elements, for example, if the New Card Toast is still displayed and I tap on it, it is said correctly via voice-over.

So based on the above yes, there is an attempt of New Card Saved to be said via Voice Over but not fully.

!Video (14).mov|width=220,height=480!

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v122 (37590) with iPhone 15 Pro (17.1.2).
